web/ui/react-app/public/index.html

```diff
@@ -37,7 +37,11 @@
       work correctly both with client-side routing and a non-root public URL.
       Learn how to configure a non-root public URL by running `npm run build`.
     -->
-    <title>Prometheus Expression Browser</title>
+    <!--
+        The TITLE_PLACEHOLDER magic value is replaced during serving by Prometheus.
+        We need it dynamic because it can be overrided by the command line flag web.page-title.
```

Comment:
```suggestion
        We need it dynamic because it can be overriden by the command line flag --web.page-title.
```

Reply:
I am afraid -- will not play nicely with all the browser (my editor is complaining). Picked an alertnative syntax and fixed the typo.